lib/server/handlers/sthRemoveDataHandler.js

```diff
@@ -64,7 +64,10 @@ function removeDataHandler(request, reply) {
         },
         function(err) {
             if (err) {
-                if (err.name === 'MongoError' && err.message.indexOf('does not exist. Currently in strict mode')) {
+                if(err.name === 'MongoNetworkError' && err.message.includes('failed to connect to server')){
+                    sthLogger.error(request.sth.context,'Database is not connected');
+                    response =  reply(err);
+                } else if (err.name === 'MongoNetworkError' && err.message.includes('does not exist. Currently in strict mode')) {
```

Comment:
```suggestion
                } else if (err.name === 'MongoError' && err.message.includes('does not exist. Currently in strict mode')) {
```

To keep the same checking that before the PR

(The change from .indexOf to .includes in the other part of the "if" seems to be a good improvement)

Reply:
Done in commit https://github.com/telefonicaid/fiware-sth-comet/pull/574/commits/f27942ab13c3bea324146b846eeb00fbac9f68b3